src/ui/server/helpers/summary-lists/your-business/index.ts

```diff
@@ -21,11 +26,16 @@ const generateFields = (answers: ApplicationExporterCompany, referenceNumber: nu
  * yourBusinessSummaryList
  * Create multiple groups with govukSummaryList data structure
  * @param {ApplicationExporterCompany} answersExporterCompany application exporterCompany object
- * @param {Number} referenceNumber application reference number
+ * @param {ApplicationExporterBusiness} anwersExporterBusiness application exporterBusiness object
+ * @param {Number} referenceNumber Application reference number
```

Comment:
Typo from last PR 😛

Reply:
finally found the typo

Reply:
It's still there 😂 should just be 

```js
Application reference number
```


Not a blocker